booking-up-for-beauty/smunilla/booking_up_for_beauty.go

```diff
@@ -0,0 +1,57 @@
+package booking
+
+import (
+	"fmt"
+	"time"
+)
+
+func parseDate(date string) time.Time {
+	var layouts = []string{
+		"1/2/2006 15:04:05",
+		"January 2, 2006 15:04:05",
+		"Monday, January 2, 2006 15:04:05",
+	}
+
+	for _, layout := range layouts {
+		t, err := time.Parse(layout, date)
+		if err == nil {
+			return t
+		}
+	}
+
+	panic(fmt.Sprintf("Problem parsing date: %s", date))
+}
+
+// Schedule returns a time.Time from a string containing a date
+func Schedule(date string) time.Time {
+	return parseDate(date)
+}
+
+// HasPassed returns whether a date has passed
+func HasPassed(date string) bool {
+	return time.Now().After(parseDate(date))
+}
+
+// IsAfternoonAppointment returns whether a time is in the afternoon
+func IsAfternoonAppointment(date string) bool {
+	apptHour := parseDate(date).Hour()
+
+	return apptHour >= 12 && apptHour <= 18
```

Comment:
Small typo not covered by the tests I guess:
```suggestion
	return apptHour >= 12 && apptHour < 18
```

Reply:
Good catch! If this were real, we'd also update the tests now :)